.github/workflows/pypi_upload.yml

```diff
@@ -0,0 +1,31 @@
+name: pypi_upload
```

Comment:
doesn't seem like this belongs in this PR?

Reply:
Yeah I’ll rebase before I merge ...